sql-parser/src/main/java/io/crate/sql/tree/BooleanLiteral.java

```diff
@@ -35,9 +37,10 @@
     public BooleanLiteral(String value)
```

Comment:
I think we could make the ctor private to enforce the use of the two singletons (TRUE_LITERAL and FALSE_LITERAL) (and then change the paramter to the ctor to boolean value)